### DISCUSSION TITLE
Once live citation refreshed in Word, would it be converted back to keys?

### DISCUSSION THREAD

**Discussion by jasonandjack1997** [retorquere/zotero-better-bibtex#2675] [General]
As my coauthors use different editors I need to share between them. However once the @plain text citations in mardown files converted to docx via pandoc and zotero lua filter, and then refreshed in Word, then I found I had no ways to convert the docx back to md without losing the keys. 

Is there anyway to solve this?

**Comment by retorquere** [comment]
Only [sort of](https://retorque.re/zotero-better-bibtex/citing/migrating/).

**Reply by jasonandjack1997** [reply]
I have to say you are a genius retorquere!

**Reply by retorquere** [reply]
I wouldn't go that far 😆 but I'll take the compliment.
